version.tf

```diff
@@ -6,5 +6,10 @@ terraform {
       source  = "IBM-Cloud/ibm"
       version = ">= 1.56.1, < 2.0.0"
     }
+    # tflint-ignore: terraform_unused_required_providers
+    time = {
+      source  = "hashicorp/time"
+      version = ">= 0.9.1, < 1.0.0"
+    }
```

Comment:
why add this? It was just removed in a recent PR

Reply:
Removed it

Reply:
@iamar7 I still see it?